test/script/reference.t

```diff
@@ -13,7 +17,11 @@
   $ owi script --no-exhaustion reference/const.wast
   $ owi script --no-exhaustion reference/conversions.wast
   $ owi script --no-exhaustion reference/custom.wast
+  expected unexpected end but there was no error
+  [7]
   $ owi script --no-exhaustion reference/data.wast
+  expected unknown memory 0 but there was no error
+  [7]
```

Comment:
This is because we don't validate simplified modules (see the comment in `script.ml`), as validation is currently done while doing the text to simplified transformation. We should split them in the future.